lib/literal.rb

```diff
@@ -19,6 +19,52 @@ module Literal
 		loader.setup
 	end
 
+	CLASS_METHOD = Kernel.instance_method(:class)
+	CLASS_VARIABLE_GET_METHOD = Module.instance_method(:class_variable_get)
+	INSTANCE_VARIABLE_GET_METHOD = Kernel.instance_method(:instance_variable_get)
+
+	module BindingAssert
+		def assert(...)
+			Literal.assert(self, ...)
+		end
+	end
+
+	::Binding.include(Literal::BindingAssert)
```

Comment:
We should make a `literal/pure` entry point that people can use if they want to opt out of patches.

Reply:
Elaborate? It has no perf cost, no overhead. And this DX is superior to `Literal.assert(binding, **constraints)`.